packages/office-ui-fabric-react/src/components/Button/examples/Button.Split.Example.tsx

```diff
@@ -14,7 +14,7 @@ export class ButtonSplitExample extends React.Component<IButtonProps> {
     const { disabled, checked } = this.props;
 
     const getClassNames = classNamesFunction<IButtonBasicExampleStyleProps, IButtonBasicExampleStyles>();
-    const classNames = getClassNames(getStyles);
+    const classNames = getClassNames(getStyles, {});
```

Comment:
I don't think that `{}` is needed - the definition of getClassNames already defaults to `{}` for you...

Reply:
Yes, initially I didn't allow it but David decided to keep it optional for backwards compatibility. The original version would have been undefined. My POV is that if you don't need style props, you should just pass in a style object and so all style functions should take in the 2nd arg.